src/Dfe.PlanTech.Web/ProgramExtensions.cs

```diff
@@ -136,7 +137,13 @@ public static IServiceCollection AddCaching(this IServiceCollection services)
 
     public static IServiceCollection AddDatabase(this IServiceCollection services, IConfiguration configuration)
     {
-        void databaseOptionsAction(DbContextOptionsBuilder options) => options.UseSqlServer(configuration.GetConnectionString("Database"));
+        void databaseOptionsAction(DbContextOptionsBuilder options) => options.UseSqlServer(configuration.GetConnectionString("Database"),
+        opts =>
+        {
+            var databaseRetryOptions = configuration.GetSection("Database").Get<DatabaseOptions>();
+            opts.EnableRetryOnFailure(databaseRetryOptions.MaxRetryCount, TimeSpan.FromMilliseconds(databaseRetryOptions.MaxDelayInMilliseconds), null);
+            opts.UseQuerySplittingBehavior(QuerySplittingBehavior.SplitQuery);
+        });
```

Comment:
whats the impact (if any) of this option?

Reply:
Generally improved performance for larger queries, [Microsoft's documentation on it](https://learn.microsoft.com/en-us/ef/core/querying/single-split-queries) will cover it better than I could.

I've removed it though as I didn't mean to add it to be honest. Although we did have it originally (at least on the CmsDbContext). It shouldn't make a difference on any of the requests we have currently anyway.